docs/styles/style.css

```diff
@@ -0,0 +1,95 @@
+* {
+  margin: 0;
+}
+
+body {
+  font-size: 16px;
+  font-family: "Roboto", Arial, sans-serif;
+  line-height: 1.5; /* baseline height 24p x*/
+}
+
+p {
+  font-size: 1rem;
+  margin-bottom: 1.5rem;
+}
+
+p.lead {
+  font-size: 1.2rem;
+  line-height: 1.5rem;
+}
+
+li {
+  font-size: 1rem;
+}
+
+ul, ol {
+  margin-bottom: 1.5rem;
+}
+
+span {
+  font-size: 1rem;
```

Comment:
I would not define a styling for something like `span` or `div`. these tags are so common that usually you define a class if you like to style something specific. An other dev would insert a new span in the html and not expect something is changing.